Core/DISET/AuthManager.py

```diff
@@ -256,6 +262,25 @@ def getUsername(self, credDict):
       return True
     return False
 
+  def isUserSuspended(self, credDict):
+    """ Discover if the use is in Suspended status
+
+    :param dict credDict: Credentials to check
+    :return: Boolean True if user is Suspended
+    """
+    if self.KW_USERNAME not in credDict:
+      self.getUsername(credDict)
+    if self.KW_USERNAME not in credDict or self.KW_GROUP not in credDict:
```

Comment:
maybe you could add some comments on the logic ? I am not sure to understand it. 
This particular test `if self.KW_USERNAME not in credDict or self.KW_GROUP not in credDict:` means that basically if you are not presenting a user or a group, you by default are not suspended ? isn't that strange ?

Reply:
Added comments